ynh_mongo_db/ynh_mongo_db

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+
+# Create a Database in mongodb
+# Usage : ynh_mongo_setup_db dbname dbuser
+ynh_mongo_setup_db() {
+  db_name=$1
+  db_user=$2
+  db_pass=$(ynh_string_random)
+  # Create a user for the Database
+  mongo --shell "$db_name" --eval 'db.createUser( { user: "'${db_user}'", pwd: "'${db_pass}'", roles: [ "readWrite" ] } );' <<< exit
+  # Set the user as admin of the db
+  mongo --shell "$db_name" --eval 'db.grantRolesToUser("'${db_user}'",[{ role: "clusterMonitor", db: "admin" }]);' <<< exit
+  # Store Setting for the remove part.
+  ynh_app_setting_set $app dbuser $dbuser
+  ynh_app_setting_set $app dbpass $dbpass
+  ynh_app_setting_set $app dbname $dbname
+  mkdir -p /var/backups/mongobackups/
+  cat > "/etc/cron.daily/mongo_backup" << EOF
```

Comment:
Uh could you clarify the point of this cron ? Why have this backup automatically setup when one just asks for a DB to be created ? Shouldn't this be handled by a helper called from the backup scripts ? Also calling this for multiple DB's will override the previous cron :s

Reply:
yes, well... no. Because there is a backup (just one time) and a cron backup. One must be set at the install moment and another when the backup command is called.

Reply:
But normally, there will be a choice... ANyway, I wanted something in the admin interface to set up an automatic backup at any moment.